amarillo/services/secrets.py

```diff
@@ -1,10 +1,13 @@
 from typing import Dict
 from pydantic import Field
 from pydantic_settings import BaseSettings
-
+from typing import Optional
```

Comment:
could you just add this to the already existing import in line 1 (or inversely add Dict from line 1)?